frontend/Container.js

```diff
@@ -37,14 +43,42 @@ class Container extends React.Component {
     },
     extraTabs: {[key: string]: () => React$Element},
   };
+  state: State;
+
+  constructor(props: Props) {
+    super(props);
+
+    this.state = {
+      isVertical: (window.innerWidth < 500),
+    };
+  }
+
+  componentDidMount() {
+    window.addEventListener('resize', (e) => this.handleResize(e), false);
+    this.setState({
+      isVertical: (window.innerWidth < 500),
+    });
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize');
```

Comment:
I don't think you're unsubscribing since you're not passing the listener.
Instead, I suggest declaring the listener as a class property:

```js
handleResize = (e) => {
  // ...
}
```

Then you can `window.addEventListener('resize', this.handleResize)` and `window.removeEventListener('resize', this.handleResize)`.